capitolwords_ng/capitolweb/capitolweb/urls.py

```diff
@@ -15,8 +15,10 @@
 """
 from django.conf.urls import include, url
 from django.contrib import admin
+from rest_framework.documentation import include_docs_urls
 
 urlpatterns = [
+    url(r'^docs/', include_docs_urls(title='Capitol Words API')),
```

Comment:
for some reason it's not working since I added it to more than 1 place. I need to figure that out.